pkg/test/case.go

```diff
@@ -304,6 +301,18 @@ func (t *Case) CollectTestStepFiles() (map[int64][]string, error) {
 	return testStepFiles, nil
 }
 
+// getIndexFromFile returns the index derived from fileName's prefix, ex. "01-foo.yaml" has index 1.
+func (t *Case) getIndexFromFile(fileName string) (int64, bool, error) {
+	matches := testStepRegex.FindStringSubmatch(fileName)
+	if len(matches) != 2 {
+		t.Logger.Log("Ignoring", fileName, "as it does not match file name regexp:", testStepRegex.String())
+		return 0, false, nil
+	}
+
+	i, err := strconv.ParseInt(matches[1], 10, 32)
+	return i, true, err
```

Comment:
is the hasIndex really necessary? I think just returning index and error should be fine here. Now having index is an error in my mind.

Reply:
I think alena means "Not having an index is an error"... "Now" -> "Not"

I could go either way...   I don't like to introduce errors for logic and this isn't an "error".   We allow for files that are not indexed in the test folders.   They are ignored.  The log message is to bring awareness to that fact.  

The err could be a parsing error in which case we exit `CollectTestStepFiles`
In the case of an ignored file... we continue looping the rest of the files.

Reply:
I think I would prefer a default index of -1  and remove the `hasIndex`.    thus it isn't an error and we can remove the extra bool... the check external to this would be 

```
if index == -1 {
    continue
}
```